client/styles/components/_p5-contrast-codemirror-theme.scss

```diff
@@ -97,8 +97,13 @@ $p5-contrast-activeline: #999999;
 }
 
 .cm-s-p5-contrast .CodeMirror-matchingbracket {
-  outline: 1px solid $p5-contrast-darkgray;
-  color: black !important;
+  color: $p5-contrast-green !important;
```

Comment:
can you add these styles without using `!important`? in this project i prefer to not use it unless absolutely necessary.